sdv/constraints/tabular.py

```diff
@@ -221,40 +223,50 @@ class GreaterThan(Constraint):
             by checking if the value provided is a column name.
     """
 
-    _diff_column = None
+    _column_list = False
+    _diff_columns = None
     _is_datetime = None
     _column_to_reconstruct = None
 
     def __init__(self, low, high, strict=False, handling_strategy='transform',
                  fit_columns_model=True, drop=None, high_is_scalar=None,
                  low_is_scalar=None):
+        self.constraint_columns = (low, high)
+        low = [low] if isinstance(low, str) else low
```

Comment:
Following up on the comment below: This does not look right.
We should take `high` and `low` as they come and not put them inside a `list` until we have already checked that they are actual column names.

Reply:
I'm not sure, if the input is string, then they should be column names, and `_validate_constraint_columns` would handle the case if they did not exist.